indico/web/templates/forms/location_widget.html

```diff
@@ -0,0 +1,359 @@
+                    }
+                    justUse.html("Just use &quot;" + node.val() + "&quot;")
+                } else if (justUse.length != 0) {
+                    justUse.remove();
+                }
+                return resultHtmlList;
+            }
+
+            function buildBackdrop() {
+                var wrapper = $('.dropback-wrapper');
+                var wrapperCloseButton = latestUsedField.parent().find('.close-backdrop-button');
+                if (wrapper.length === 0) {
+                    var dropbackWrapper = $('<div>').attr({
+                        'class': 'dropback-wrapper'
+                    }).on('click keydown', function(e) {
+                        if (latestUsedField.val() != "") {
+                            e.stopPropagation();
+                        } else {
+                            $(this).hide();
+                            latestUsedField.parent().find('.close-backdrop-button').hide();
+                        }
+                    });
+                    var closeButton = $('<a>').attr({
+                        'class': 'icon-close close-button'
+                    }).on('click', function() {
+                        $(this).parent().hide();
+                    });
+                    dropbackWrapper.append(closeButton);
+                    latestUsedField.closest('.i-location-field').append(dropbackWrapper);
+                    wrapperCloseButton.show();
+                } else {
+                    wrapper.show();
+                    wrapperCloseButton.show();
+                }
+            }
+
+            function postSelectionActions(inputField, value) {
+                inputField.val(value);
+                field.val(JSON.stringify(hiddenData));
+                inputField.trigger('indico:close-results-list');
+            }
+
+            function highlightOption(inputField, value) {
+                var highlighter = inputField.parent().find('.highlighter');
+                highlighter.html(value).addClass('visible');
+            }
+
+            function cleanQuery(query) {
+                if (query) {
+                    query = query.trim();
+                    if (!query) {
+                        query = ''
+                    }
+                }
+                return query
+            }
+
+            usingDefault.on('click', function() {
+                toggleDefaultLocation($(this).prop('checked'));
+            });
+
+            toggleDefaultLocation(usingDefault.prop('checked'));
+
+            venueInput.typeahead({
+                source: venues,
+                minLength: 0,
+                searchOnFocus: true,
+                resultContainer: '#' + venueInput.siblings('.typeahead-results').find('.results-list-container').attr('id'),
+                template: {% raw %}'<span>{{name}}</span>'{% endraw %},
+                emptyTemplate: function (query) {
+                    return "No results for \"" + query + "\"";
```

Comment:
Thing is that this message is never displayed. However, I need to return a string in order to create the "Just use" option. I didn't know what to put so I added this copying it from the docs somewhere. I could return an empty string if you think it's cleaner.